lib/models.js

```diff
@@ -62,6 +62,10 @@ class DataElementSpecifications {
     return this.all.filter(de => de.isEntry);
   }
 
+  get abstracts() {
+    return this.all.filter(de => de.isAbstract);
+  }
```

Comment:
I'll remove it for now, and if a need ever comes up, it'll be simple enough to add back.